documentation/book/assembly-custom-resources.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-custom-resources-{context}']
+= Custom resources
+Custom Resource Definitions (CRDs) extend the Kubernetes API, providing definitions to create and add custom resources to an {ProductPlatformName} cluster. Custom resources are created as instances of CRDs.
+
+For {Productname}, CRDs introduce custom resources specific to {Productname} to an {ProductPlatformName} cluster, such as Kafka, Kafka Connect, Kafka Mirror Maker, and users and topics.  CRDs provide configuration instructions, defining the schemas used to instantiate and manage the resources specific to {Productname}. CRDs also allow {Productname} resources to benefit from native {ProductPlatformName} features like CLI accessibility and configuration validation.
+
+CRDs require a one-time installation to a cluster. Depending on the cluster setup, installation typically requires cluster admin privileges.
+
+NOTE: `ClusterRoles` and `ClusterRoleBindings` to manage the CRDs are applied on installation to provide access for the Cluster Operator.
+
+CRDs and custom resources are defined as YAML files. Separate CRD files contain specifications for creating specific custom resources for {Productname}.
+
+Custom resources are instantiated and validated using the specifications defined in a CRD.
+
+WARNING: When CRDs are deleted, custom resources of that type are also deleted.
```

Comment:
In all the above you've not really mentioned/made concrete how a CR relates to its CRD. The CRD defines a new **kind** of resource within a Kubernetes cluster. That definition means the API server will let users create resources with that kind (literally, `kind: Kafka` etc), and means the API server knows how to validate such resources.

Reply:
We have this now:

A CRD defines a new `kind` of resource — such as `kind:kafka` — within a {ProductPlatformName} cluster.

The {ProductPlatformName} API server allows custom resources to be created based on the `kind` and understands from the CRD how to validate and store the custom resource when it is added to the {ProductPlatformName} cluster.